src/poker/game/GamePlay.java

```diff
@@ -160,342 +162,329 @@ public void inPositionPlayerCanCheckPostFlop(Player player1, Player player2) {
         System.out.println("1 = call 2 = raise 3 = fold");
         Scanner scanner = new Scanner(System.in);
         int decision2 = scanner.nextInt();
         if (decision2 == 1) {
-            upadePotStackAmountInHandSmallBlindPlayer(amountToCallSmallBlind(player1, player2), player2);
+            System.out.println(player2.getNamePlayer() + " calls " + amountToCall(player1,player2));
+            upadePotStackAmountInHandPlayer(amountToCall(player1, player2), player2);
             makeBoardDependsGamePhase();
         } else if (decision2 == 2) {
-            makeRaiseSmallBlind(player1, player2, amountToCallSmallBlind(player1, player2));
-            outPositionPlayerGotBetOrRaisePostFlop(player1, player2);
+            makeRaiseSmallBlind(player1, player2, amountToCall(player1, player2));
+            postFlopActionBigGotBetOrRaise(player1, player2,amountToCall(player1,player2));
         } else {
             System.out.println(player2.getNamePlayer() + " folds");
             System.out.println(player1.getNamePlayer() + " wins " + gameState.getPot() + "$");
         }
     }
 
-    //1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
-//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
-//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
-//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
-//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
-//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
-//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
-//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
-//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
-//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
-//1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
-    public void preFlopSmallBlind(Player player1, Player player2) {
-        double raise = 0;
-        System.out.println(player1.getNamePlayer() + " bigBlind " + bigBlind + "$");
-        System.out.println(player2.getNamePlayer() + " smallBlind " + smallBlind + "$");
-        printPot();
-        System.out.println(player2.getNamePlayer() + " podejmuje decyzje");
-        howMuchToCallSmallBlind(player1, player2);
-        System.out.println("1 = call 2 = raise");
+    public void preflopActionSmall(Player player1, Player player2) {
+        System.out.println("Zaczyna " + player2.getNamePlayer());
+        preFlopDetails(player1,player2);
```

Comment:
zobacz stworzyles funckje preFlopDetails(...) ktora PRINTUJE jakies rzeczy zeby bylo to wydzielone do funkcji, a wyzej dales System.out.println("Zaczyna " + player2.getNamePlayer()); gdzie to tak naprawde tez powinno byc w funkcji ktora ma za zadanie printowac jakies rzeczy. Wiec to tez trzeba wlozyc to do preFlopDetails(...)